bin/fuel-core/src/cli/run/graphql.rs

```diff
@@ -112,6 +112,14 @@ pub struct QueryCosts {
     )]
     pub dry_run: usize,
 
+    /// Query costs for generating execution trace for a block.®
+    #[clap(
+        long = "query-cost-storage-read-replay",
+        default_value = DEFAULT_QUERY_COSTS.storage_read_replay.to_string(),
+        env
+    )]
+    pub storage_read_replay: usize,
```

Comment:
The verbiage around this isn't clear to me. 

This argument is for setting the costs? I think more context is required here and perhaps a more specific name for the field.

Reply:
Yes. It's a filed in a struct called `QueryCosts`, and all other arguments have similar purpose.

Reply:
Ah ah ah. I remember this now. I suggested on another PR that we should rename this to "Complexity" or something (just because we already have concepts of gas costs etc).  I won't nitpick too much on this since at least it's within the `graphql` crate, but giving some clarity in the comment on what types of costs this is would be valuable (even if the other args aren't that clear).